frontend/.eslintrc

```diff
@@ -154,7 +154,7 @@
                         "ignoreRegExpLiterals": true
                     }
                 ],
-                "indent": ["error", 4]
+                "indent": ["error", 4, { "SwitchCase": 1 }]
```

Comment:
Do not change elsint

Reply:
How I can resolve this eslint-prettier stupid error ?
https://prnt.sc/uYvf_dTCX0Xu